### PR TITLE
fixes in opentelemetry tags

### PR DESCRIPTION
**Pull request by fjod**
https://github.com/fsharplang-ru/pulsar-client-dotnet/issues/191

image

### CODE REVIEW

**Comment by fjod** [comment]
image

File: src/Pulsar.Client.Otel/OTelProducerInterceptor.fs [hunk 1]
```diff
@@ -43,9 +43,9 @@ type OTelProducerInterceptor<'T>(sourceName: string, log: ILogger) =
                 message    //If there are no listeners interested in this activity, the activity above will be null <..> Ensure that all subsequent calls using this activity are protected with a null check.                
             else
                 activity
+                    .SetTag("messaging.system", "pulsar") 
                     .SetTag("messaging.destination_kind", "topic") 
-                    .SetTag("messaging.destination", producer.Topic) 
-                    .SetTag("messaging.operation", "Produce")
```
Comment (Lanayx) [inline src/Pulsar.Client.Otel/OTelProducerInterceptor.fs:48]: I think "messaging.operation" should be named as "send" here. I believe we should also add "producer_id" to correspond new producer tag.

Comment (fjod) [reply src/Pulsar.Client.Otel/OTelProducerInterceptor.fs:48]: Activity.Id:          00-fda0a095dec6b7262be9a9908ac0910f-0baedaaed0e2a3b6-01
Activity.ActivitySourceName: pulsar.producer
Activity.DisplayName: persistent://public/default/my-topic-637703405384905191 send
Activity.Kind:        Producer
Activity.StartTime:   2021-10-20T12:28:59.0930771Z
Activity.Duration:    00:00:00.0332724
Activity.TagObjects:
    messaging.system: pulsar
    messaging.destination_kind: topic
    messaging.destination: persistent://public/default/my-topic-637703405384905191
    **messaging.operation: send**
    **messaging.producer_id: 0**
    acknowledge.type: Success
    messaging.message_id: 727:0:-1:0
Resource associated with Activity:
    service.name: telemetry
    service.instance.id: 4ed4f62b-3a20-42d7-9abb-98a725fd2521

Activity.Id:          00-1f6d5051f8cbb6bc4276521ccd10f8af-aa99dea00a423100-01
Activity.ActivitySourceName: pulsar.producer
Activity.DisplayName: persistent://public/default/my-topic-637703405384905191 send
Activity.Kind:        Producer
Activity.StartTime:   2021-10-20T12:28:59.1016436Z
Activity.Duration:    00:00:00.0618826
Activity.TagObjects:
    messaging.system: pulsar
    messaging.destination_kind: topic
    messaging.destination: persistent://public/default/my-topic-637703405384905191
    messaging.operation: send
    messaging.producer_id: 0
    acknowledge.type: Success
    messaging.message_id: 727:1:-1:0
Resource associated with Activity:
    service.name: telemetry
    service.instance.id: 4ed4f62b-3a20-42d7-9abb-98a725fd2521

Sent 1 from F# at 20.10.2021 15:28:59
Sent 2 from F# at 20.10.2021 15:28:59
Activity.Id:          00-fda0a095dec6b7262be9a9908ac0910f-6ff4e788753a8ea9-01
Activity.ParentId:    00-fda0a095dec6b7262be9a9908ac0910f-0baedaaed0e2a3b6-01
Activity.ActivitySourceName: pulsar.consumer
Activity.DisplayName: persistent://public/default/my-topic-637703405384905191 receive
Activity.Kind:        Consumer
Activity.StartTime:   2021-10-20T12:28:59.1742030Z
Activity.Duration:    00:00:00.0079655
Activity.TagObjects:
    messaging.system: pulsar
    messaging.destination_kind: topic
    messaging.destination: persistent://public/default/my-topic-637703405384905191
    **messaging.consumer_id: 0**
    messaging.message_id: 727:0:-1:0
    **messaging.operation: receive**
    acknowledge.type: CumulativeAck
Resource associated with Activity:
    service.name: telemetry
    service.instance.id: 4ed4f62b-3a20-42d7-9abb-98a725fd2521

Activity.Id:          00-1f6d5051f8cbb6bc4276521ccd10f8af-bc83b1e09d6aecee-01
Activity.ParentId:    00-1f6d5051f8cbb6bc4276521ccd10f8af-aa99dea00a423100-01
Activity.ActivitySourceName: pulsar.consumer
Activity.DisplayName: persistent://public/default/my-topic-637703405384905191 receive
Activity.Kind:        Consumer
Activity.StartTime:   2021-10-20T12:28:59.1768948Z
Activity.Duration:    00:00:00.0079679
Activity.TagObjects:
    messaging.system: pulsar
    messaging.destination_kind: topic
    messaging.destination: persistent://public/default/my-topic-637703405384905191
    messaging.consumer_id: 0
    messaging.message_id: 727:1:-1:0
    messaging.operation: receive
    acknowledge.type: CumulativeAck
Resource associated with Activity:
    service.name: telemetry
    service.instance.id: 4ed4f62b-3a20-42d7-9abb-98a725fd2521

Comment (fjod) [reply src/Pulsar.Client.Otel/OTelProducerInterceptor.fs:48]: After adding these tags ^

**Pull request merged**